src/nectarchain/dqm/start_calib.py

```diff
@@ -16,11 +16,27 @@
 from trigger_statistics import TriggerStatistics
 from camera_monitoring import CameraMonitoring
 
+import argparse
+
+# Create an ArgumentParser object
+parser = argparse.ArgumentParser(description='My script')
+parser.add_argument('-plot', action='store_true', help='Whether to plot the data')
```

Comment:
The standard usage for optional arguments is `-<1 CHARACTER>` and/or `--<STRING>` (note the double dash), so I would suggest:
```
parser.add_argument('-p', '--plot',
                                        action='store_true',
                                        help='Enables plots to be generated')
```

Reply:
okay will be in the next commit